sites/docs/src/content/docs/tutorials/tests_and_test_data/components/03_testing_modules.md

```diff
@@ -0,0 +1,734 @@
+│                                                                                                        │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+INFO     Generating nf-test snapshot again to check stability
+╭──────────────────────────────────────────── nf-test output ────────────────────────────────────────────╮
+│                                                                                                        │
+│ 🚀 nf-test 0.9.0                                                                                       │
+│ https://www.nf-test.com                                                                                │
+│ (c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr                                                   │
+│                                                                                                        │
+│ Load .nf-test/plugins/nft-bam/0.5.0/nft-bam-0.5.0.jar                                                  │
+│ Load .nf-test/plugins/nft-compress/0.1.0/nft-compress-0.1.0.jar                                        │
+│ Load .nf-test/plugins/nft-vcf/1.0.7/nft-vcf-1.0.7.jar                                                  │
+│ Load .nf-test/plugins/nft-csv/0.1.0/nft-csv-0.1.0.jar                                                  │
+│ Load .nf-test/plugins/nft-utils/0.0.3/nft-utils-0.0.3.jar                                              │
+│ Load .nf-test/plugins/nft-fastq/0.0.1/nft-fastq-0.0.1.jar                                              │
+│ Load .nf-test/plugins/nft-anndata/0.1.0/nft-anndata-0.1.0.jar                                          │
+│                                                                                                        │
+│ Test Process ABRICATE_SUMMARY                                                                          │
+│                                                                                                        │
+│   Test [fc133477] 'Should run without failures' PASSED (11.658s)                                       │
+│                                                                                                        │
+│                                                                                                        │
+│ SUCCESS: Executed 1 tests in 11.667s                                                                   │
+│                                                                                                        │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+INFO     All tests passed!
+```
+
+### Global `setup` method
+
+A global `setup` method can be defined for all tests within a `nextflow_process` definition. The `setup` is applied to multiple `test` cases, ensuring a consistent setup for each test. This approach is useful when multiple tests share the same setup requirements.
```

Comment:
```suggestion
A global `setup` method can be defined for all tests within a `nextflow_process` definition.
The `setup` is applied to multiple `test` cases, ensuring a consistent setup for each test.
This approach is useful when multiple tests share the same setup requirements.
```

Reply:
Maybe it's worht adding an admoniton that it doesn't mean the setup process is only run once, and the output is shared across all tests, it just means you write less duplicated code - actually teh setup block is run for each test execution that requires it (a common misconception)